packages/ui/src/atom/label-atom/label-atom.vue

```diff
@@ -2,6 +2,7 @@
   <label
     class="label-atom"
     :class="themeCssClass"
+    :disabled="disabled"
```

Comment:
The same question. `disabled` attribute is not specific for non-interactive elements, like `span`, `label` etc. Pass it only to inputs, buttons and etc.

Reply:
But if you need to add some styles - use additional class instead of this attribute.

Reply:
On `label` disabled may be really required, if `label` binded to the input inside or via `for` property. Here, I guess, some styles denends of that attribute. @dudiq am I right?

Reply:
disabled on label may be need if label binded to input via for attribute. Maybe here tooltip in label disabling via `disabled` attr.